ui/src/message_popup/playtest/video_alpha_experience_page.jsx

```diff
@@ -0,0 +1,263 @@
+/* @flow weak */
+import _ from 'lodash';
+import React from 'react';
+import uuid from 'uuid';
+import VelocityTransitionGroup from "velocity-react/velocity-transition-group";
+
```

Comment:
This is pretty much a copy of `danson_experience_page.jsx` (i.e., the version that existed about  a month ago). I simply added webcam recording at the top of the intro page.